modules/@apostrophecms/login/views/login.html

```diff
@@ -3,6 +3,14 @@
 
 {% block bodyClass %}apos-login-page{% endblock %}
 
+{% block beforeMain %}
+  {# Hush up project level styles while login loads #}
+  <style>
+    body {
+      background-color: var(--a-background-primary) !important;
```

Comment:
😱

Reply:
ELAINE: "So, this is over, not over? I'm betting not over."